docs/fault-proof-alpha/manual.md

```diff
@@ -0,0 +1,110 @@
+
+The creator of the game selects the output root to dispute. It is identified by its L2 block number which can be used to
+look up the full details in the L2 output oracle.
+
+The agreed output root is defined as the output root immediately prior to the disputed output root in the L2 output
+oracle. Therefore, a dispute game should only be created for the first invalid output root. If it is successfully
+disputed, all output roots after it would be considered invalid by inference.
+
+The L1 head block can be any L1 block where the disputed output root is present in the L2 output oracle. Proposers
+should therefore ensure that all batch data has been submitted to L1 before submitting a proposal. The L1 head block is
+recorded in the `BlockOracle` and then referenced by its block number.
+
+Creating a game requires two separate transactions. First the L1 head block is recorded in the `BlockOracle` by calling
+its `checkpoint` function. This record the parent of the block the transaction is included in. The `BlockOracle` emits a
+log `Checkpoint(blockNumber, blockHash, childTimestamp)`.
+
+Now the L1 head, along with the information on the output roots from the L2 output oracle can be used to execute cannon
+and determine the root claim to use when creating the game. In simple cases, where the claim is expected to be
+incorrect, an arbitrary hash can be used for claim values. For more advanced cases [cannon can be used](./cannon.md) to
+generate a trace, including the claim values to use at specific steps. Note that it is not valid to create a game that
+disputes an output root, using the final hash from a trace that confirms the output root is valid. To dispute an output
+root successfully, the trace must resolve that the disputed output root is invalid.
+
+The game can then be created by calling the `create` method on the `DisputeGameFactory` contract. This requires three
+parameters:
+
+- `gameType` - a `uint8` representing the type of game to create. For fault dispute games using cannon and op-program
+  traces, the game type is 0.
+- `rootClaim` - a `bytes32` hash of the final state from the trace.
+- `extraData` - arbitrary bytes which are used as the initial inputs for the game. For fault dispute games using cannon
+  and op-program traces, this is the abi encoding of `(uint256(l2_block_number), uint256(l1_checkpoint))`.
+    - `l2_block_number` is the L2 block number from the output root being disputed
+    - `l1_checkpoint` is the L1 block number recorded by the `BlockOracle` checkpoint
+
+This emits a log event `DisputeGameCreated(gameAddress, gameType, rootClaim)` where `gameAddress` is the address of the
+newly created dispute game.
+
+A simple helper script, [create_game.sh](../../op-challenger#create_gamesh) has been created to easily create a new
+dispute game and also provides an example of using `cast` to manually create a game.
```

Comment:
Adopted the first part of this change, but not sure that `includes` is a better word than `provides` here - makes it sound like there's an add-on part of the script that's the example of using `cast` rather than the script essentially being that example.

Have changed it to be `and also acts as an example` which I think might be better....